alf/experience_replayers/replay_buffer.py

```diff
@@ -466,6 +517,51 @@ def _pad(self, x, env_ids):
         return ((self._current_pos[env_ids] - x - 1) //
                 self._max_length) * self._max_length + x
 
+    def _set_default_return(self, env_ids):
+        mask = torch.ones_like(env_ids, dtype=torch.bool)
+        if torch.any(mask):
+            current_pos = self._current_pos[env_ids]
+            current_pos -= 1
+            ind = (env_ids[mask], self.circular(current_pos[mask]))
```

Comment:
Isn't env_ids[mask] same as env_ids since mask is an all-one tensor?
Same question for current_pos[mask]